### ISSUE TITLE
Fatal Error- Unable to reactivate Woocommerce

### ISSUE THREAD

**Issue by heather-ferguson** [woothemes/woocommerce#33611]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

I received notification from Wordpress that they caught a fatal error with Woocommerce. Woocommerce and all extensions were deactivated. When I try to reactivate woocommerce, I receive a message that the plugin could not be activated because it caused a fatal error. Below is the latest up to date information for what my site is using. 

WordPress version 6.0
Active theme: OceanWP (version 2.0.7)
Current plugin: WooCommerce (version 6.5.1)
PHP version 7.4.28

**Comment by jonathansadowski** [comment]
Hi @heather-ferguson,

Thanks for the report. Is there any additional information available with the fatal error? Are you still unable to activate WooCommerce with other plugins disabled? Have you tried activating WooCommerce with the latest version (6.6.1)?

**Comment by heather-ferguson** [comment]
Wordpress sent me the following information on June 16, 2022. My Store page has been down ever since. When I attempt to load Woocommerce (Version 6.6.1), I receive a message that states they are unable to activate this plugin because it triggered a fatal error.


When seeking help with this issue, you may be asked for some of the following information:
WordPress version 6.0
Active theme: OceanWP (version 2.0.7)
Current plugin: WooCommerce (version 6.5.1)
PHP version 7.4.28

Error Details
=============
An error of type E_COMPILE_ERROR was caused in line 102 of the file /home4/fergust1/public_html/wp-content/plugins/woocommerce/src/Container.php. Error message: Declaration of Automattic\WooCommerce\Container::has($id) must be compatible with Psr\Container\ContainerInterface::has(string $id): bool

**Comment by deleriouseel** [comment]
I am having the same issue when I updated to 6.6.1. I narrowed it down to a conflict with VaultPress. If VaultPress is activated I get a fatal error. If WooCommerce is activated I get a critical error. I tried rolling back to 6.6.0 but no change.

Fatal Error in the debug.log:
[29-Jun-2022 17:01:47 UTC] PHP Fatal error:  Declaration of Automattic\WooCommerce\Container::has($id) must be compatible with Psr\Container\ContainerInterface::has(string $id): bool in /home1/northdi7/public_html/staging/8523/wp-content/plugins/woocommerce/src/Container.php on line 104

WordPress version 6.0
Active theme: X-Child Theme 1.0.0, X-Parent theme 9.1.4
Current plugin: WooCommerce (version 6.6.1), VaultPress 2.2.1
PHP version 7.4.30

**Comment by jonathansadowski** [comment]
Thanks for the updates @heather-ferguson  and @deleriouseel.

What were the last versions of WooCommerce that you were able to activate? Are you still able to activate them? If you are able to activate a previous version, could you share your [system status report](https://woocommerce.com/document/understanding-the-woocommerce-system-status-report/)?

The System Status Report is found in your WordPress admin under **WooCommerce > Status**. 
Please select “Get system report”, then “Copy for support”, and then paste it in reply to this thread.

**Comment by heather-ferguson** [comment]
(version 6.5.1) was the last version that was used. It is not currently activated and the new version can not be activated because of the fatal error. So I currently have no version activated and therefore can not retrieve the status report.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by deleriouseel** [comment]
My last version was 6.6.0. I rolled back to it and it also triggered the fatal error, so I have 6.6.1 installed now. 

`
### WordPress Environment ###

WordPress address (URL): https://northcountrychapel.com/staging/8523
Site address (URL): https://northcountrychapel.com/staging/8523
WC Version: 6.6.1
REST API Version: ✔ 6.6.1
WC Blocks Version: ✔ 7.6.2
Action Scheduler Version: ✔ 3.4.0
Log Directory Writable: ✔
WP Version: 6.0
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache
PHP Version: 7.4.30
PHP Post Max Size: 516 MB
PHP Time Limit: 60
PHP Max Input Vars: 1000
cURL Version: 7.83.1
OpenSSL/1.1.1p

SUHOSIN Installed: –
MySQL Version: 5.7.23-23
Max Upload Size: 512 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 6.6.1
WC Database Prefix: staging_wp_
Total Database Size: 167.11MB
Database Data Size: 133.96MB
Database Index Size: 33.15MB
staging_wp_woocommerce_sessions: Data: 0.01MB + Index: 0.00MB + Engine MyISAM
staging_wp_woocommerce_api_keys: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_woocommerce_attribute_taxonomies: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_woocommerce_downloadable_product_permissions: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_woocommerce_order_items: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
staging_wp_woocommerce_order_itemmeta: Data: 0.05MB + Index: 0.04MB + Engine MyISAM
staging_wp_woocommerce_tax_rates: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
staging_wp_woocommerce_tax_rate_locations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_woocommerce_shipping_zones: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_woocommerce_shipping_zone_locations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_woocommerce_shipping_zone_methods: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_woocommerce_payment_tokens: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_woocommerce_payment_tokenmeta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_woocommerce_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
directory: Data: 0.26MB + Index: 0.02MB + Engine MyISAM
directory_2category: Data: 0.02MB + Index: 0.02MB + Engine MyISAM
directory_category: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
products: Data: 0.01MB + Index: 0.00MB + Engine MyISAM
products_categories: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_actionscheduler_actions: Data: 0.02MB + Index: 0.03MB + Engine MyISAM
staging_wp_actionscheduler_claims: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_actionscheduler_groups: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
staging_wp_actionscheduler_logs: Data: 0.02MB + Index: 0.02MB + Engine MyISAM
staging_wp_bsearch: Data: 0.02MB + Index: 0.03MB + Engine MyISAM
staging_wp_bsearch_daily: Data: 0.04MB + Index: 0.04MB + Engine MyISAM
staging_wp_commentmeta: Data: 0.13MB + Index: 0.03MB + Engine MyISAM
staging_wp_comments: Data: 0.19MB + Index: 0.06MB + Engine MyISAM
staging_wp_links: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_options: Data: 2.83MB + Index: 0.35MB + Engine MyISAM
staging_wp_postmeta: Data: 8.30MB + Index: 2.19MB + Engine MyISAM
staging_wp_posts: Data: 7.23MB + Index: 8.03MB + Engine MyISAM
staging_wp_revslider_css: Data: 0.09MB + Index: 0.00MB + Engine MyISAM
staging_wp_revslider_css_bkp: Data: 0.09MB + Index: 0.00MB + Engine MyISAM
staging_wp_revslider_layer_animations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_revslider_layer_animations_bkp: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_revslider_navigations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_revslider_navigations_bkp: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_revslider_sliders: Data: 0.02MB + Index: 0.00MB + Engine MyISAM
staging_wp_revslider_sliders_bkp: Data: 0.03MB + Index: 0.00MB + Engine MyISAM
staging_wp_revslider_slides: Data: 0.12MB + Index: 0.00MB + Engine MyISAM
staging_wp_revslider_slides_bkp: Data: 0.21MB + Index: 0.00MB + Engine MyISAM
staging_wp_revslider_static_slides: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_revslider_static_slides_bkp: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_termmeta: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
staging_wp_terms: Data: 0.06MB + Index: 0.11MB + Engine MyISAM
staging_wp_term_relationships: Data: 0.41MB + Index: 0.74MB + Engine MyISAM
staging_wp_term_taxonomy: Data: 0.06MB + Index: 0.08MB + Engine MyISAM
staging_wp_usermeta: Data: 0.06MB + Index: 0.05MB + Engine MyISAM
staging_wp_users: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
staging_wp_wc_admin_notes: Data: 0.04MB + Index: 0.00MB + Engine MyISAM
staging_wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.01MB + Engine MyISAM
staging_wp_wc_category_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_wc_customer_lookup: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
staging_wp_wc_download_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_wc_order_coupon_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_wc_order_product_lookup: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
staging_wp_wc_order_stats: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
staging_wp_wc_order_tax_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_wc_product_attributes_lookup: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
staging_wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
staging_wp_wc_product_meta_lookup: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
staging_wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
staging_wp_wc_reserved_stock: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_wc_tax_rate_classes: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
staging_wp_wc_webhooks: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_woocommerce_termmeta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_wpbackitup_job_control: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
staging_wp_wpbackitup_job_items: Data: 8.02MB + Index: 0.00MB + Engine InnoDB
staging_wp_wpbackitup_job_tasks: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
staging_wp_wpmailsmtp_debug_events: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
staging_wp_wpmailsmtp_tasks_meta: Data: 0.03MB + Index: 0.01MB + Engine MyISAM
staging_wp_wpml_mails: Data: 1.52MB + Index: 0.00MB + Engine InnoDB
staging_wp_wpsc_also_bought: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_wpsc_cart_contents: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_wpsc_cart_item_meta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_wpsc_checkout_forms: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
staging_wp_wpsc_claimed_stock: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_wpsc_coupon_codes: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_wpsc_currency_list: Data: 0.02MB + Index: 0.01MB + Engine MyISAM
staging_wp_wpsc_download_status: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_wpsc_meta: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
staging_wp_wpsc_product_rating: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_wpsc_purchase_logs: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_wpsc_purchase_meta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_wpsc_region_tax: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_wpsc_submited_form_data: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
staging_wp_wpsc_visitors: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
staging_wp_wpsc_visitor_meta: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
studies: Data: 3.80MB + Index: 0.03MB + Engine MyISAM
studies_books: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
studies_categories: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_actionscheduler_actions: Data: 0.03MB + Index: 0.03MB + Engine MyISAM
wp_actionscheduler_claims: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_actionscheduler_groups: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wp_actionscheduler_logs: Data: 0.02MB + Index: 0.02MB + Engine MyISAM
wp_bsearch: Data: 0.02MB + Index: 0.04MB + Engine MyISAM
wp_bsearch_daily: Data: 0.04MB + Index: 0.04MB + Engine MyISAM
wp_commentmeta: Data: 0.22MB + Index: 0.04MB + Engine MyISAM
wp_comments: Data: 0.20MB + Index: 0.07MB + Engine MyISAM
wp_links: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_options: Data: 4.73MB + Index: 0.39MB + Engine MyISAM
wp_postmeta: Data: 25.17MB + Index: 2.56MB + Engine MyISAM
wp_posts: Data: 12.21MB + Index: 16.52MB + Engine MyISAM
wp_revslider_css: Data: 0.09MB + Index: 0.00MB + Engine MyISAM
wp_revslider_css_bkp: Data: 0.09MB + Index: 0.00MB + Engine MyISAM
wp_revslider_layer_animations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_revslider_layer_animations_bkp: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_revslider_navigations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_revslider_navigations_bkp: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_revslider_sliders: Data: 0.02MB + Index: 0.00MB + Engine MyISAM
wp_revslider_sliders_bkp: Data: 0.03MB + Index: 0.00MB + Engine MyISAM
wp_revslider_slides: Data: 0.12MB + Index: 0.00MB + Engine MyISAM
wp_revslider_slides_bkp: Data: 0.21MB + Index: 0.00MB + Engine MyISAM
wp_revslider_static_slides: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_revslider_static_slides_bkp: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_termmeta: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wp_terms: Data: 0.06MB + Index: 0.14MB + Engine MyISAM
wp_term_relationships: Data: 0.41MB + Index: 0.79MB + Engine MyISAM
wp_term_taxonomy: Data: 0.06MB + Index: 0.08MB + Engine MyISAM
wp_usermeta: Data: 0.07MB + Index: 0.05MB + Engine MyISAM
wp_users: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wp_wc_admin_notes: Data: 0.04MB + Index: 0.00MB + Engine MyISAM
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.01MB + Engine MyISAM
wp_wc_category_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wc_customer_lookup: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wp_wc_download_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wc_order_coupon_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wc_order_product_lookup: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
wp_wc_order_stats: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wp_wc_order_tax_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wc_product_attributes_lookup: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wc_tax_rate_classes: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wp_wc_webhooks: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_api_keys: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_attribute_taxonomies: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_downloadable_product_permissions: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_order_itemmeta: Data: 0.05MB + Index: 0.04MB + Engine MyISAM
wp_woocommerce_order_items: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
wp_woocommerce_payment_tokenmeta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_payment_tokens: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_sessions: Data: 0.14MB + Index: 0.01MB + Engine MyISAM
wp_woocommerce_shipping_zones: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_shipping_zone_locations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_shipping_zone_methods: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_tax_rates: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wp_woocommerce_tax_rate_locations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_termmeta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wpbackitup_job_control: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_wpbackitup_job_items: Data: 55.63MB + Index: 0.00MB + Engine InnoDB
wp_wpbackitup_job_tasks: Data: 0.08MB + Index: 0.00MB + Engine InnoDB
wp_wpmailsmtp_debug_events: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpmailsmtp_tasks_meta: Data: 0.03MB + Index: 0.01MB + Engine MyISAM
wp_wpml_mails: Data: 0.13MB + Index: 0.00MB + Engine InnoDB
wp_wpsc_also_bought: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wpsc_cart_contents: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wpsc_cart_item_meta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wpsc_checkout_forms: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wp_wpsc_claimed_stock: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wpsc_coupon_codes: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wpsc_currency_list: Data: 0.02MB + Index: 0.01MB + Engine MyISAM
wp_wpsc_download_status: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wpsc_meta: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wp_wpsc_product_rating: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wpsc_purchase_logs: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wpsc_purchase_meta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wpsc_region_tax: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wpsc_submited_form_data: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wpsc_visitors: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wp_wpsc_visitor_meta: Data: 0.00MB + Index: 0.01MB + Engine MyISAM

### Post Type Counts ###

attachment: 655
custom_css: 1
infobox: 4
jp_img_sitemap: 2
jp_sitemap: 4
jp_sitemap_index: 2
jp_sitemap_master: 2
jp_vid_sitemap: 2
mc4wp-form: 2
nav_menu_item: 40
page: 48
post: 3932
product: 11
product_variation: 8
revision: 2632
shop_order: 61
shop_order_refund: 1
sidebar: 3
slide: 30
staging_wp_global_st: 1
testimonial: 12
wooframework: 208
wpcf7_contact_form: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (28) ###

Akismet Anti-Spam: by Automattic – 4.2.4
Async JavaScript: by Frank Goossens (futtta) – 2.21.08.31
Author Category: by Bainternet – 0.8
Better Search: by Ajay D'Souza – 3.1.0
Bluehost: by Bluehost – 2.10.1
Classic Editor: by WordPress Contributors – 1.6.2
Classic Widgets: by WordPress Contributors – 0.3
Contact Form 7: by Takayuki Miyoshi – 5.6
Cornerstone: by Themeco – 6.1.4
Hello Dolly: by Matt Mullenweg – 1.7.2
Jetpack Boost: by Automattic - Website Speed and Performance team – 1.5.1
Jetpack: by Automattic – 11.0
MC4WP: Mailchimp for WordPress: by ibericode – 4.8.7
Category Order: by Wessley Roche – 1.0.3
Really Simple CAPTCHA: by Takayuki Miyoshi – 2.1
Really Simple CSV Importer: by Takuro Hishikawa – 1.3
Reveal IDs: by Oliver Schlöbe – 1.5.4
WooCommerce Stripe Gateway: by WooCommerce – 6.4.2
WooCommerce Shipping & Tax: by WooCommerce – 1.26.1
WooCommerce: by Automattic – 6.6.1
WP BackItUp Premium: by WPBackItUp – 1.25.0
WP BackItUp Community Edition: by WPBackItUp – 1.42
WP Mail Logging: by Wysija – 1.10.4
WP Mail SMTP: by WPForms – 3.4.0
WP Rollback: by Impress.org – 1.7.1
WP-Sweep: by Lester 'GaMerZ' Chan – 1.1.5
X – Google Analytics: by Themeco – 1.0.3
X – Typekit: by Themeco – 1.0.3

### Inactive Plugins (4) ###

Custom Post Order: by OLT UBC – 1.1
MailChimp: by MailChimp – 1.5.7
VaultPress: by Automattic – 2.2.1
WordPress Importer: by wordpressdotorg – 0.7

### Settings ###

API Enabled: ✔
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –
Enforce Approved Product Download Directories: ✔

### WC Pages ###

Shop base: #22534 - /shop/
Cart: #22621 - /shop/cart/
Checkout: #22623 - /shop/checkout/
My account: #22625 - /shop/my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: X – Child Theme
Version: 1.0.0
Author URL: http://theme.co/
Child Theme: ✔
Parent Theme Name: X
Parent Theme Version: 9.1.4
Parent Theme Author URL: https://theme.co
WooCommerce Support: ✔

### Templates ###

Overrides: x-child/cart/cross-sells.php
x-child/single-product-reviews.php


### Admin ###

Enabled Features: activity-panels
analytics
coupons
customer-effort-score-tracks
experimental-products-task
experimental-import-products-task
experimental-fashion-sample-products
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
wc-pay-promotion
wc-pay-welcome-page
wc-pay-subscriptions-page

Disabled Features: minified-js
settings

Daily Cron: ✔ Next scheduled: 2022-06-30 01:44:39 -07:00
Options: ✔
Notes: 40
Onboarding: completed

### Action Scheduler ###

Complete: 95
Oldest: 2022-06-26 08:09:52 -0700
Newest: 2022-06-29 10:14:00 -0700

Pending: 2
Oldest: 2022-06-30 08:25:31 -0700
Newest: 2022-07-04 14:00:00 -0700


### Status report information ###

Generated at: 2022-06-29 12:24:14 -07:00
`

**Comment by Konamiman** [comment]
Hi @heather-ferguson, we want to fix this issue but so far we've been unable to reproduce it. The url you provided seems to suggest that it's a staging site; would it be possible for you to create a user for us so that we can log in and take a look? If possible it would be great if we could get SSH as well, since we'll probably need to try a few tweaks on the WooCommerce code. Thank you very much!

cc @peterfabian and @barryhughes

**Comment by heather-ferguson** [comment]
Website: https://www.fergusonlearning.com

I have created a user ***@***.******@***.***, you should receive a separate email but I have included the password below.

*(pw removed by @barryhughes)*

I am not sure what SSH is or how to get you access to it.

Just FYI- My site is currently live and is not a staging site.

If you could provide instructions for your SSH request, I can try to do it. I apologize I am not tech-savvy. 😊

**Comment by heather-ferguson** [comment]
I just made a huge mistake! I was trying to delete the site that was the staging site. When I deleted the staging site both listings disappeared and my site has now disappeared. Can you please help me access the backup so I can get my site back up. At this point I can see nothing! Please help!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Heather Ferguson
Sent: Friday, July 1, 2022 7:13:41 AM
To: woocommerce/woocommerce ***@***.***>
Subject: RE: [woocommerce/woocommerce] Fatal Error- Unable to reactivate Woocommerce (Issue #33611)


Website: https://www.fergusonlearning.com



I have created a user ***@***.******@***.***>, you should receive a separate email but I have included the password below.



ep$Bz5y3%TYfKgwd#qH2lANq



I am not sure what SSH is or how to get you access to it.



Just FYI- My site is currently live and is not a staging site.



If you could provide instructions for your SSH request, I can try to do it. I apologize I am not tech-savvy. ??

**Comment by barryhughes** [comment]
Sorry to hear that!

However, we don't have any access to your site or to your backups: I'd recommend reaching out to whichever company provides that service for you (and if you use a specific plugin to manage backups, they might have documentation you can refer to).

**Comment by barryhughes** [comment]
In case handy, one way to replicate essentially the same problem is to create a single-file plugin as follows:

```php
<?php
/**
 * Plugin name: Test #33611
 */

namespace Psr\Container {
	/**
	 * This is a condensed version of the same interface found in php-fig/container 2.0.0.
	 * 
	 * @see https://github.com/php-fig/container/blob/2.0.0/src/ContainerInterface.php
	 */
	interface ContainerInterface {
		public function get(string $id);
		public function has(string $id): bool;
	}
}
```

This simulates a plugin that includes `psr/container:2.0.0` as one of its dependencies. From there:

* Deactivate all plugins.
* Activate the test plugin.
* Activate WooCommerce, see fatal error.

<img width="924" alt="psr-container-fatal" src="https://user-images.githubusercontent.com/3594411/177767436-a724ad35-50f3-4f3b-b29d-5d3704251b9f.png">

**Comment by heather-ferguson** [comment]
I switched server providers (no longer with bluehost) and magically woocommerce is working again.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by peterfabian** [comment]
I believe this should be fixed now that #33703 is merged.
